src/test/regress/sql/pg14.sql

```diff
@@ -78,6 +78,23 @@ CALL citus_cleanup_orphaned_shards();
 SELECT result AS column_compression FROM run_command_on_workers($$SELECT ARRAY(
 SELECT attname || ' ' || attcompression FROM pg_attribute WHERE attrelid::regclass::text LIKE 'pg14.col\_compression%' AND attnum > 0 ORDER BY 1
 )$$) ORDER BY length(result);
+-- ALTER STATISTICS .. OWNER TO CURRENT_ROLE
+CREATE TABLE st1 (a int, b int);
+CREATE STATISTICS role_s1 ON a, b FROM st1;
+SELECT create_distributed_table('st1','a');
+ALTER STATISTICS role_s1 OWNER TO CURRENT_ROLE;
 
+CREATE ROLE role_1 WITH LOGIN SUPERUSER;
+SELECT run_command_on_workers($$CREATE ROLE role_1 WITH LOGIN SUPERUSER;$$);
+SET ROLE role_1;
+ALTER STATISTICS role_s1 OWNER TO CURRENT_ROLE;
+\c - - - :worker_1_port
+SELECT rolname FROM pg_roles WHERE oid IN (SELECT stxowner FROM pg_statistic_ext WHERE stxname LIKE 'role\_s1%');
```

Comment:
why do we only run this on worker 1? Can we use run_command_on_workers? That way we won't need to connect to `worker1_port` which resets connection settings(such as we need to set the search path again)

Reply:
makes sense, updated